mythril/laser/ethereum/svm.py

```diff
@@ -107,12 +108,25 @@ def __init__(
         self._start_sym_exec_hooks = []  # type: List[Callable]
         self._stop_sym_exec_hooks = []  # type: List[Callable]
 
+        self._start_exec_hooks = []  # type: List[Callable]
+        self._stop_exec_hooks = []  # type: List[Callable]
+
         self.iprof = iprof
         self.instr_pre_hook = {}  # type: Dict[str, List[Callable]]
         self.instr_post_hook = {}  # type: Dict[str, List[Callable]]
         for op, _, _, _ in OPCODES.values():
             self.instr_pre_hook[op] = []
             self.instr_post_hook[op] = []
+        self.hook_type_map = {
```

Comment:
Yeah, that's perfect

Reply:
Actually we might end up needing string constants, so there wouldn't be any change there

Reply:
There are two ways:-
Method 1:- current way, we use self._something_hooks variables at the needed locations
Method 2:- remove the self._something_hooks, so we might need to access them with self.hook_types[some_string], since it's a string which will be used in many places we might need to add a bunch of string constants, so there wouldn't be any change in LoC.
But maybe Method 2 looks cleaner.